venue/views.py

```diff
@@ -1190,6 +1202,15 @@ def confirm_email_change(request):
         token = AuthToken.objects.get(salt=code)
         new_email = rtemp.retrieve(code)
         if code and new_email:
+            # Update the email in Constant Contact
+            resp = get_contstant_contact_record(token.user.email)
+            if resp.status_code == 200 and len(resp.json()['results']) > 0:
+                cc_id = resp.json()['results'][0]['id']
+                update_constant_contact_email(
```

Comment:
If the user is not found in Constant Contact, it seems like the user should be added, not that the update should be skipped.